js/main.js

```diff
@@ -18,4 +19,91 @@ $entryForm.addEventListener('submit', function submitForm(event) {
   data.nextEntryId++;
   $img.setAttribute('src', './images/placeholder-image-square.jpg');
   $entryForm.reset();
+
+  const $newDomTree = renderEntry(entryData);
+  const $unorderedList = document.querySelector('ul');
+  $unorderedList.prepend($newDomTree);
+  viewSwap('entries');
+  toggleNoEntries();
+});
+
+function renderEntry(entry) {
+  const $listItem = document.createElement('li');
+  $listItem.setAttribute('class', 'row');
+
+  const $columnHalf1 = document.createElement('div');
+  $columnHalf1.setAttribute('class', 'column-half');
+  $listItem.appendChild($columnHalf1);
+
+  const $img = document.createElement('img');
+  $img.setAttribute('src', entry.imageUrl);
+  // $img.setAttribute('src', 'https://archives.bulbagarden.net/media/upload/4/4a/0025Pikachu.png');
+  $columnHalf1.appendChild($img);
+
+  const $columnHalf2 = document.createElement('div');
+  $columnHalf2.setAttribute('class', 'column-half');
+
+  $listItem.appendChild($columnHalf2);
+
+  const $h3Title = document.createElement('h3');
+  $h3Title.textContent = entry.title;
+  $columnHalf2.appendChild($h3Title);
+
+  const $pNotes = document.createElement('p');
+  $pNotes.textContent = entry.notes;
+  $columnHalf2.appendChild($pNotes);
+
+  return $listItem;
+}
+
+document.addEventListener(
+  'DOMContentLoaded',
+  function handleDOMContentLoaded(event) {
+    let $journalEntry;
+    const $unorderedList = document.querySelector('ul');
```

Comment:
Once you move the other `querySelector` to the global space, this won't be needed